### PR TITLE
Blog post for v3.0.0-rc1 release

### LINKED ISSUE #5907: 🎉 Release for Sprint 223 (v3.0.0-rc1) 🎉
Issue to track work for publishing a new release of `odo` at the end of the Sprint.

- Version: **v3.0.0-rc1**
- Target release process start date: **Aug 29, 2022**

Note: Write an announcement blog post once the release is done.

### PR DESCRIPTION
**Pull request by feloy**
**What type of PR is this:**

/kind documentation
Fixes #5907

### CODE REVIEW

**Comment by valaparthvi** [comment]
The blog lgtm, can you also update the installation docs, or it's fine by me if you want to take this in a different PR?

File: docs/website/blog/2022-09-01-odo-v3-rc1.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+## Note about Release Candidate
+
+With rc1, we are feature complete with the following exceptions:
+- odo list services ([\#5996](https://github.com/redhat-developer/odo/pull/5996))
+- ability to turn off auto restart and trigger sync manually in odo dev ([\#5634](https://github.com/redhat-developer/odo/pull/5634))
+
+Our GA release will include:
+- Onboarding updates, including updates to the Quickstart guide
+- Video tutorial
```
Comment (valaparthvi) [inline docs/website/blog/2022-09-01-odo-v3-rc1.md:111]: This reads a little vague to me. Can you add more specific comments or remove this altogether?

**Review by valaparthvi — CHANGES_REQUESTED** [review]

**Review by valaparthvi — CHANGES_REQUESTED** [review]
There's one last place you need to change and that's docusaurus-config.js. It currently points to Beta2.

**Review by rm3l — CHANGES_REQUESTED** [review]
Can you also update the version label in [`docusaurus.config.js`](https://github.com/redhat-developer/odo/blob/main/docs/website/docusaurus.config.js#L130) ? I think it was not updated for the Beta 3 release.

File: docs/website/blog/2022-09-01-odo-v3-rc1.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+**Merged pull requests:**
+
+- Bump to version 3.0.0-rc1 [\#6064](https://github.com/redhat-developer/odo/pull/6064) ([feloy](https://github.com/feloy))
+- Update script to support alpha/beta/rc versions [\#6063](https://github.com/redhat-developer/odo/pull/6063) ([feloy](https://github.com/feloy))
+- update to OpenShift 4.11 [\#6060](https://github.com/redhat-developer/odo/pull/6060) ([kadel](https://github.com/kadel))
+- Troubleshoot IBM Cloud Windows tests failing [\#6038](https://github.com/redhat-developer/odo/pull/6038) ([feloy](https://github.com/feloy))
+- test: use `T.Setenv` to set env vars in tests [\#6037](https://github.com/redhat-developer/odo/pull/6037) ([Juneezee](https://github.com/Juneezee))
+- Revert "Add blog post for v3.0.0-beta3 and update installation docs" [\#6030](https://github.com/redhat-developer/odo/pull/6030) ([cdrage](https://github.com/cdrage))
+
+## Note about Release Candidate
```
Comment (rm3l) [inline docs/website/blog/2022-09-01-odo-v3-rc1.md:103]: Maybe this section could be the first one, before listing the notable changes? It currently looks a bit "hidden" to me..

**Comment by rm3l** [comment]
> There's one last place you need to change and that's docusaurus-config.js. It currently points to Beta2.

Ah, I didn't see this comment before adding a similar one :-)

File: docs/website/blog/2022-09-01-odo-v3-rc1.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/aWxb_GXLW_w" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+`odo list` combines the output of `odo list component` and `odo list binding`, for human-readable and JSON output.
+
+#### odo no longer requires the optional 'metadata.name' field in Devfile ([#6015](https://github.com/redhat-developer/odo/pull/6015))
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/20QOOAwil8M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+If the `metadata.name` field is present in the Devfile, it is used as name of the deployed component. If not present, the directory name is used instead.
```
Comment (rm3l) [inline docs/website/blog/2022-09-01-odo-v3-rc1.md:55]: This needs to be clarified, I think. If `metadata.name` is not present, it uses Alizer to detect name. If not possible, the directory name is used. So maybe we could write it like so:

```suggestion
If the `metadata.name` field is present in the Devfile, it is used as name of the deployed component. If not present, odo tries to detect the name automatically based on the project configuration. If this is not possible, the directory name is used instead.
```

**Pull request merged**